base/src/main/java/org/geysermc/api/util/ApiVersion.java

```diff
@@ -0,0 +1,91 @@
+package org.geysermc.api.util;
+
+/**
+ * Represents a version of the api.
+ */
+public class ApiVersion {
```

Comment:
```suggestion
/**
 * Represents a version of an api.
 */
public class ApiVersion {
```

because it may be used for Base API or Geyser API or something else that extends the Base API

Reply:
As described in the Geyser PR, commit the geyser versioning document here and link to it here